omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/DrugOrderSubclassHandler1_8.java

```diff
@@ -149,4 +150,20 @@ public static String getDisplay(DrugOrder delegate) {
 		// TODO dates, etc
 		return ret.toString();
 	}
+	
+	@PropertyGetter("auditInfo")
+	public SimpleObject getAuditInfo(DrugOrder delegate) throws Exception {

```

Comment:
How comes here you are not delegating to the method in OrderResource? Like you did for TestOrderSubclassHandler1_10?